packages/valory/services/oracle_polygon/service.yaml

```diff
@@ -10,14 +10,15 @@ fingerprint_ignore_patterns: []
 agent: valory/oracle:0.1.0:bafybeifperyoibdy3fitngdpr4ou6ig5t5xflje7g4dgcajuzwbymzxb64
 number_of_agents: 4
 ---
-setup_args:
+params_args:
```

Comment:
Why this change?

Reply:
Ahh sorry, forgot to comment. These are not anymore only for setup arguments, but also for other parameters.